src/services/affinidi/bff-service.ts

```diff
@@ -169,6 +176,11 @@ export class BFFService {
     const res = await instance.post('/api/login/import-user-groups', { data }, { headers })
     return res.data as any
   }
+
+  public async generateKeyPair(): Promise<{ publicKey: KeyLike; privateKey: KeyLike }> {
+    const { publicKey, privateKey } = await generateKeyPair('PS256')
```

Comment:
Can we use the value from [SupportedAlgorithms](https://github.com/affinidi/affinidi-cli/blob/main/src/common/constants.ts#L2C12-L2C17)?